.pipelines/e2e-with-billing.yml

```diff
@@ -17,7 +17,7 @@ stages:
         subscription: $(e2e-subscription)
         azureDevOpsE2EJSONSPN: $(aro-v4-e2e-devops-spn)
         aroVersionStorageAccount: $(aro-version-storage-account)
-        rpMode: int
+        rpMode: $(RP_MODE)
```

Comment:
I not just drop it at all. we already know no mode -> prod. Why overspecify in the first place ?
And you know you can test this PR by going into pipeline in question and run it with Branch:`refs/pull/1500/merge` ? As I did it here https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=42609183&view=results

Reply:
I like deleting it as best idea. 

I thought the pipeline itself ran off master against code base at particular commit. Did not know ADO runs pipeline as specified on branch. Trying to learn how to use ADO :sweat_smile:

Reply:
Ignore by suggesion. this breaks in pipeline :/